CondTools/RunInfo/interface/BTransitionAnalyzer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef BTRANSITIONANALYZER_H
+#define BTRANSITIONANALYZER_H
+
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CondCore/CondDB/interface/Utils.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+
+namespace cond {
+  template<class T, class R>
+  class BTransitionAnalyzer: public edm::one::EDAnalyzer<edm::one::WatchRuns, edm::one::SharedResources> {
+  public:
+    BTransitionAnalyzer( const edm::ParameterSet& pset ):
+      m_currentThreshold( pset.getUntrackedParameter<double>( "currentThreshold", 18000. ) ) {
+      usesResource( "PoolDBOutputService" );
```

Comment:
@diguida I have just one comment/question. You declare that you use the resource "PoolDBOutputService". As I understood, this is only needed, if the resource is not thread-safe. However, @Dr15Jones commented in [this HN thread](https://hypernews.cern.ch/HyperNews/CMS/get/edmFramework/3665/1/1/1/1/1.html) that it appears to be thread-safe.
Do you have reasons to declare it anyways, except that "appears to be thread-safe" might be a bit vague?